lib/ffprober/ffmpeg/finder.rb

```diff
@@ -0,0 +1,23 @@
+module Ffprober
+  module Ffmpeg
+    class Finder
+      def self.path
+        @path ||= begin
+          path = ENV['PATH'].split(File::PATH_SEPARATOR).find do |path_to_check|
+            File.executable?(File.join(path_to_check, executable_name))
+          end
+
+          path && File.expand_path(executable_name, path)
+        end
+      end
+
+      def self.executable_name
+        @executable_name ||= self.windows? ? 'ffprobe.exe' : 'ffprobe'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.